examples/trigger-events/versions.tf

```diff
@@ -2,10 +2,6 @@ terraform {
   required_version = ">= 0.15.0"
 
   required_providers {
-    random = {
-      source  = "hashicorp/random"
-      version = ">= 3.1.0"
-    }
```

Comment:
Not sure we want to remove this, benchmarks is not part of `trigger-events`, so this provider probably shouldn't change